hedgehog/hedgehog.cabal

```diff
@@ -75,6 +75,7 @@ library
     , time                            >= 1.4        && < 1.10
     , transformers                    >= 0.4        && < 0.6
     , transformers-base               >= 0.4        && < 0.5
+    , unordered-containers
```

Comment:
Not sure what version to set here yet - I guess travis will show us what works for all GHCs?

Reply:
Resolved this in latest fpush